lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -239,6 +239,23 @@ class ConvertTorchConstantIntOp
 };
 } // namespace
 
+namespace {
+class ConvertAtenIntBoolOp : public OpConversionPattern<AtenIntBoolOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenIntBoolOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Type resultType =
+        this->getTypeConverter()->convertType(op->getResult(0).getType());
+    Value result =
+        convertScalarToDtype(rewriter, op.getLoc(), adaptor.getA(), resultType);
```

Comment:
oh nice. really wish we have a glossary of helper functions.

Reply:
I just stole this from surrounding code. Useful but not always obvious when to use.